custom/live-streaming/components/LiveStreamingModal.js

```diff
@@ -18,35 +18,53 @@ const LAYOUTS = [
   { label: 'Active participant', value: 'active-participant' },
 ];
 
+const ParticipantOption = ({ sessionId }) => {
+  const participant = useParticipant(sessionId);
+
+  return (
+    <option value={participant.session_id} key={sessionId}>
+      {participant.user_name}
```

Comment:
same here, it might be worth adding a null check since `useParticipant()` can return `null`